src/services/GearScout-Service.tsx

```diff
@@ -0,0 +1,31 @@
+import { IMatch, IUser } from '../model/Models.ts';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+type GearscoutResponse<T> = Promise<Response>;
+
+class GearscoutService {
+	private baseURL: string = 'https://data.gearitforward.com/';
+
+	submitMatch = (teamNumber: IUser, secretCode: IUser, match: IMatch): GearscoutResponse<void> => {
```

Comment:
By specifying a type of `IUser`, we're asking for an object of user info. So, we can replace `teamNumber: IUser, secretCode: IUser` with just `user: IUser`.

Then in the code below, we would use `user.teamNumber` and `user.secretCode` to get the right information.